spec/models/subscriber_list_spec.rb

```diff
@@ -29,6 +29,46 @@
     end
   end
 
+  describe "validations" do
+    subject { FactoryGirl.build(:subscriber_list) }
```

Comment:
Can do. I'm somewhat indifferent. `subject` is supposed to be the main object that you're testing (system under test).